commands/group.js

```diff
@@ -154,6 +156,36 @@ module.exports = {
 			await bot.db.updateGroup(msg.author.id, group.name, "description", description.slice(0,2000));
 			if(description.length > 2000) return "Description updated, but was cut to 2000 characters to fit within Discord embed limits.";
 			return "Description updated.";
+			
+		case "purge":
+			if(!args[1]) return "No group name given.";
+			if(args[1] == "*") {
+				try {
+				await bot.send(msg.channel, `Warning: This will remove the group along with all of the ${cfg.lang}s within it. Reply 'yes' to continue or anything else to cancel.`);
```

Comment:
This warning message should be edited to reflect that `tul!group purge *` will remove absolutely everything - all members and all groups

Reply:
oh shit does it remove ALL of them? I thought I made it so that it only removed the one group and all members in it

Reply:
this is within the `tul!group purge *` case which would be a command to "purge all groups", the code for purging a single group is down below

Reply:
oh wait duh lol Im dumb and have no reading comprehension